### PR TITLE
Ohm update read the docs

### PR DESCRIPTION
**Pull request by rmrlangford**
Update to Setup and configuration.md page

### CODE REVIEW

File: docs/setup-and-configuration.md [hunk 1]
```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
...
+
+### How to add clients
+> **Note**: All fields marked with a * or ** indicates a mandatory field.
+
+    * - Indicates a required field which means that it cannot be left blank
+    ** - Indicates that one of the fields are required
+
+1. Logon to your OpenHIM console
+1. Click on Clients found in the left navigation menu.
+1. Click on the button labelled “+ Client” to open a popup window where you will be able to supply the client details.
```
Comment (MattyJ007) [inline docs/setup-and-configuration.md]: There is an inconsistency within the different lists - some lists elements end in full stops some don't. Please choose one style and apply to all the lists

File: docs/setup-and-configuration.md [hunk 2]
```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
...
+1. Click on Clients found in the left navigation menu.
+1. Locate the client to be removed and click on the red X button on the far right
+1. You will be prompted to confirm your action to delete the chosen client
+1. Click on the Delete button
+
+> **Note**: This action automatically deletes the role if the role was created primarily for the client that is busy being deleted with no other clients sharing the same role.
+
+### How to edit clients
+
+1. Logon to your OpenHIM console
```
Comment (MattyJ007) [inline docs/setup-and-configuration.md]: Logon is not a term used on the console - please use either Sign in or Log in for all occurences

File: docs/setup-and-configuration.md [hunk 3]
```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
...
+1. Click on Clients found in the left navigation menu.
```
Comment (MattyJ007) [inline docs/setup-and-configuration.md]: When referring to specific buttons you sometimes use double quotes around them and sometimes not - please choose one style for the document

File: docs/setup-and-configuration.md [hunk 4]
```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
```
Comment (MattyJ007) [inline docs/setup-and-configuration.md]: Please remove the blank line at the beginning of this file - it should be the main heading

Comment (MattyJ007) [reply docs/setup-and-configuration.md]: I missed it on previous review

File: docs/setup-and-configuration.md [hunk 5]
```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
...
+
+The following is an explanation of the fields that are used in the Add Client form:
+- **Client ID** - This is a unique ID which a client will use as a reference when adding channels as well as for authorisation checking.
+- **Client Name** - This is a descriptive name of the client.
+- **Domain** - A domain that is associated with a client. 
+    > **Note**: The domain needs to match the CN of the certificate if a certificate is used otherwise the client won’t be authorised successfully
+- **Roles** - The client roles field is a list of authorized user groups that are allowed to access this channel. You can either select a role from the suggested roles that appear when you start typing, or you can add a new role to the list by typing in the role and pressing “Enter“. 
+    > **Note**: suggested roles will only appear as you type, if they already exist in the OpenHIM.
+- **Certificate** - The certificate field is used when the OpenHIM core is running using mutual TLS authentication and needs to authenticate requests coming from the client. 
+By default, the OpenHIM core uses mutual TLS authentication
```
Comment (MattyJ007) [inline docs/setup-and-configuration.md]: Please move this - It should not be on its own line

File: docs/setup-and-configuration.md [hunk 6]
```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
...
+    ** - Indicates that one of the fields are required
+
+1. Logon to your OpenHIM console
+1. Click on Clients found in the left navigation menu.
+1. Click on the button labelled “+ Client” to open a popup window where you will be able to supply the client details.
+1. Capture the client details
+1. Assign an existing role or enter a name for a new role which will be created and linked to this client
+1. You may choose to make use of a basic auth password or client certificate, depending on your OpenHIM configuration. If basic auth is enabled in the OpenHIM core configuration, then only a password is required and if mutual TLS authentication is enabled, then only a client certificate is required:
+
+    -  *Certificate*: You may choose one of the available client certificates from the Client Certificate drop down.
```
Comment (MattyJ007) [inline docs/setup-and-configuration.md]: Please remove spaces from between the '-' and *Certificate*

File: docs/setup-and-configuration.md [hunk 7]
```diff
@@ -11,5 +11,486 @@ A client is usually some system that you want to able to send request to the Ope
...
+1. Click on the amber button that looks like a pencil on the far right
+1. Update the client information as required
+1. Click on the Save Changes button to update the client.
+
+### Client Roles
+
+The purpose of these roles is to act as a list of authorised user groups which are allowed to access and use a given channel. These roles are generally assigned during the creation process when adding a new client. 
+
+The following rules apply to roles:
+- A role may be assigned to one more more clients.
```
Comment (MattyJ007) [inline docs/setup-and-configuration.md]: Please add a blank line before the list starts

**Comment by rmrlangford** [comment]
Happy with changes

**Pull request merged**